dali/c_api/c_api.cc

```diff
@@ -25,13 +25,15 @@ void daliCreatePipeline(daliPipelineHandle* pipe_handle,
     int length,
     int batch_size,
     int num_threads,
-    int device_id) {
+    int device_id,
+    int pipeline_length) {
```

Comment:
That does not seem like a good name. prefetch_queue_depth perhaps?

Reply:
Done